src/test/common/fdc3.app-channels.ts

```diff
@@ -126,9 +126,9 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
       it(acTestId7, async () => {
         const errorMessage = `\r\nSteps to reproduce:\r\n- App A retrieves an app channel\r\n- App A adds a context listener of type fdc3.instrument\r\n- App B retrieves a different app channel\r\n- App B broadcasts a context of type fdc3.instrument${documentation}`;
 
-        const testChannel = await cc.createTestChannel("a-different-test-channel");
-        cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
-        cc.openChannelApp(acTestId7, undefined, APP_CHANNEL_AND_BROADCAST)
+        const testChannel = await cc.createTestChannel();
+        await cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
```

Comment:
Adding wrong listener type. Should fdc3.instrument according to test definition.